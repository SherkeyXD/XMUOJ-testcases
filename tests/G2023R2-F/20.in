783598 830550
