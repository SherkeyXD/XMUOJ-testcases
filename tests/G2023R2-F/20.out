3690545448622141
